Dockerfile

```diff
@@ -14,11 +13,21 @@ ENV NODE_ENV=production
 COPY . .
 RUN npm run build
 
-# The same image but now only install the production dependencies as the frontend is already built using vite in the build-stage
-FROM gcr.io/distroless/nodejs22-debian12@sha256:b765815eafacee5222bfa50179028f41dd8c642b68ad68ec4e6922d3b1ff2710 AS production
+# Remove dev dependencies
```

Comment:
I was confused why you would clean up because you take a new base image in l21 anyway. I guess you are removing the dev dependencies from node modules since the node modules will be copied in l30?
Maybe add a short note in the comment as it is uncommon to remove things when the next thing is a new base image :)

Reply:
Added a comment